_layouts/default.html

```diff
@@ -32,5 +35,12 @@
         {% include footer.html %}
         </div>
       </div>
+        
+        
+    <script>
```

Comment:
same goes for this, this can be at the bottom of https://github.com/mozilla/fx-webcompat-campaign/blob/master/_layouts/home.html#L30

Is it possible to get this file from a CDN? If so, we could add like we add tabletop here: https://github.com/mozilla/fx-webcompat-campaign/blob/master/_layouts/home.html#L29